specs/wp-signup-spec.js

```diff
@@ -175,8 +175,15 @@ describe( `[${ host }] Sign Up  (${ screenSize }, ${ locale })`, function() {
 			return await ReaderPage.Expect( driver );
 		} );
 
-		step( 'Can delete our newly created account', async function() {
-			return await new DeleteAccountFlow( driver ).deleteAccount( blogName );
+		after( 'Can delete our newly created account', async function() {
```

Comment:
Can these be simplified by putting the catch in deleteAccount?

Reply:
I just noticed that taking it out of deleteAccount was part of the changes. Is there a specific reason for that?

Reply:
@bsessions85 Basically, it's because of cases like [this](https://github.com/Automattic/wp-e2e-tests/pull/1780/files#diff-9bb2a6ea7acb62d90467ce1cf8ce18e6R933), when we put several different actions in `after` block. If `deleteAccount` also contains a similar message, we will have two messages if it fails. On the other hand, yes, it would simpler to put it in `deleteAccount`, and since we have only one `after` with more than `deleteAccount` I'll revert this change.